src/SplitPane.vue

```diff
@@ -38,12 +51,24 @@ function dragMove(e: MouseEvent) {
       : container.value.offsetWidth
     const dp = position - startPosition
     state.split = startSplit + +((dp / totalSize) * 100).toFixed(2)
+
+    changeViewSize()
   }
 }
 
 function dragEnd() {
   state.dragging = false
 }
+
+function changeViewSize() {
+  const el = toValue(previewRef)
+  state.viewHeight = el.offsetHeight
+  state.viewWidth = el.offsetWidth
+}
+
+defineExpose({
+  changeViewSize,
+})
```

Comment:
After the modification, it seems that it is no longer needed here.

Reply:
You're right. Fixed